src/main/python/vantiqsdk.py

```diff
@@ -1417,6 +1428,10 @@ async def connect(self, do_pings: bool = True):
                                             await callback(self.CONNECT, resp)
                                         elif request_id in self.callbacks.keys():
                                             callback = self.callbacks[request_id]
+                                            body = resp.get('body', None)
```

Comment:
This is the new protocol I've added to 1.39.  Otherwise, when a namespace goes *poof* there is no way for the client to really tell.  We also have to do this cleanup here or the client won't be able to re-subscribe should it need to (which the LCEL host does).  May want to add an explicit "unsubscribe" API, but that's outside the scope of this.